server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectionEnvironment.java

```diff
@@ -72,6 +72,17 @@ public interface GarbageCollectionEnvironment {
    */
   Stream<Reference> getReferences();
 
+  /**
+   * Return a list of all TableIDs in the
```

Comment:
It doesn't return a list. It returns a set.

Reply:
Addressed in 1f8d37305789fa1540158e32b54da86a7af3ae87